### PR TITLE
modify coreas submission scripts to work with python3

### PR DESCRIPTION
**Pull request by cg-laser**

### CODE REVIEW

**Comment by fschlueter** [comment]
Hi,
sorry for missing that. Is that ready to be merged? Is a review from my side required?

**Comment by cg-laser** [comment]
It would be great if you could have a quick look

**Comment by fschlueter** [comment]
- Is there a reason to include matplotlib in the function and not in the file header?

- You might want to set the 3 keyword of THIN. It defines a radius (in cm) around the shower axis in which particles are stored with an even higher weight in the particle file. This reduces the file size. It should have no effect on the radio emission at all and just a minor effect on the particle footprint (around the shower axis the particle density is anyway highest).

**Comment by fschlueter** [comment]
I have to admit I have never used the code to create coreas simulations. That being said it looks fine to me. (minus the comments above)

File: radiotools/coreas/geninp_aera.py [hunk 1]
```diff
@@ -24,57 +24,56 @@
...
-print "THINH   1.000E+02 1.000E+02"
-print "STEPFC ", options.stepfc
-print "OBSLEV  %.0f                       observation level (in cm)" % (float(options.obslevel) * 100)
-print "ECTMAP  1.E5                           cut on gamma factor for printout"
-print "MUADDI  T                              additional info for muons"
-print "MUMULT  T                              muon multiple scattering angle"
-print "MAXPRT  1                              max. number of printed events"
+print("ECUTS   %.4g    %.4g    %.4g     %.4g" % options.ecuts)
+print("ELMFLG  T   T                          em. interaction flags (NKG,EGS)")
+print("THIN    ", thinning, "  ", thinning * float(options.energy), " 0.000E+00")
```
Comment (fschlueter) [inline radiotools/coreas/geninp_aera.py:53]: One code add here something, see comment in PR

File: radiotools/plthelpers.py [hunk 2]
```diff
@@ -521,8 +521,7 @@ def get_histogram2d(x=None, y=None, z=None,
             z = z / np.amax(z, axis=1)[:, None]
         else:
             sys.exit("Normalisation %s is not known.")
-
-
+    import matplotlib as mpl
```
Comment (fschlueter) [inline radiotools/plthelpers.py]: Move to header?

**Review by fschlueter — APPROVED** [review]

**Comment by fschlueter** [comment]
merge?

**Pull request merged**